### PR TITLE
worked on the login form--#issue 68

### PR DESCRIPTION
**Pull request by alicendeh**
## Goal

Insert purpose of pull request

## Todos:
- [ ] Outstanding todo
- [x] Completed todo
 
## Implementation Decisions

### CODE REVIEW

**Comment by alicendeh** [comment]
Sorry it’s issue 65

File: src/components/LoginForm.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  })
+);
+const LoginForm: React.FC = () => {
+  const classes = useStyles();
+  return (
+    <div>
+      <form noValidate autoComplete='off' className={classes.form}>
+        <img src={logo} alt='Pockets Logo' className={classes.img} />
+        <TextField id='outlined-basic' label='Your Email' variant='outlined' />
+        <TextField
```
Comment (kkyeboah) [inline src/components/LoginForm.tsx]: Could you change this to a password field? 

Also, there is some alignment issue with the password text field. See screenshot below:
Screen Shot 2020-10-23 at 1 44 50 PM

File: src/components/LoginForm.tsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+      paddingTop: '1em',
+    },
+  })
+);
+const LoginForm: React.FC = () => {
+  const classes = useStyles();
+  return (
+    <div>
+      <form noValidate autoComplete='off' className={classes.form}>
+        <img src={logo} alt='Pockets Logo' className={classes.img} />
```
Comment (kkyeboah) [inline src/components/LoginForm.tsx:34]: Could you adjust the logo position to be centered? Please refer to the screenshot in the issue.

Screen Shot 2020-10-23 at 1 45 46 PM

**Review by kkyeboah — CHANGES_REQUESTED** [review]
This is a great start, just a couple of comments for improvement. Good job so far.

**Comment by alicendeh** [comment]
I have made the required changes based on @kkyeboah  review

**Comment by alicendeh** [comment]
Hi Mentor Kelvin I’ve done the corrections you asked for on the login component 
While you review that issue, please I’d like to know if am permitted to work on this one https://github.com/Pocket/curation-tools-frontend/issues/70

**Review by kkyeboah — DISMISSED** [review]
This looks good to us 👍 Nice work.

Please revert your changes to `App.tsx` and we'll be happy to re-approve and merge your PR

**Comment by alicendeh** [comment]
Hi @kkyeboah  I have reverted the change already

**Review by kkyeboah — CHANGES_REQUESTED** [review]
@alicendeh Please make sure `App.tsx` is not part of your PR changes.

**Comment by alicendeh** [comment]
I've reverted as requested is this okay to be merged
 @kkyeboah

**Comment by nina-py** [comment]
Hey @alicendeh , if you click over to the "Files changed" tab (https://github.com/Pocket/curation-tools-frontend/pull/74/files), you will see that the App.tsx file is still present in your commit with just one missing space! I know it's frustrating, but you're almost there.

Once you make one last commit to take care of that, you could also tidy up the commit history in your branch by squashing the last several commits that address the same thing - reverting the changes to App.tsx. Squashing commits is, in effect, rewriting the commit history for the branch so that there is just one commit describing the changes instead of several. There is a good guide on how to do that here: https://github.com/wprig/wprig/wiki/How-to-squash-commits.

If `git rebase -i main` doesn't work for you, try `git rebase -i HEAD~6`. The number six here is the number of commits you want to squash. The actual changes you have made to the files in those several commits will still be there.

**Comment by alicendeh** [comment]
Hi @nina-py  the problem am facing is that when I try reverting, it’s not going

**Comment by nina-py** [comment]
Don't revert then, simply do another commit. Look at the diff of App.tsx in
the "Files changed" tab - there is one missing space in the front of the
comment (highlighted in pink) and one extra space at the end of it
(highlighted in green). Fix that on your branch, commit and push your
changes, then open the "Files changed" tab on your pull request again to
make sure App.tsx has dropped off the list of files you changed.

**Pull request closed without merging**